packages/notebook/src/toc.ts

```diff
@@ -530,6 +530,9 @@ export class NotebookToCFactory extends TableOfContentsFactory<NotebookPanel> {
             ) {
               el.scrollIntoView({ block: 'center' });
             }
+          } else {
+            console.debug('scrolling to heading: using fallback strategy');
+            widget.content.scrollToItem(widget.content.activeCellIndex);
```

Comment:
`Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator`

Reply:
To provide some guide line to fix this. I suggest changing:
* `onCellInViewport` to be `async (cell: Cell): Promise<void>`
* adding `await in front of Line 535
* adding `return` at line 550 so that the catch line 552 will catch everything